### ISSUE TITLE
addUser bug

### ISSUE THREAD

**Issue by kevinmichaelchen** [TeslaGov/joan#6]
The addUser method is still throwing an error:
```json
{
  "status":"error",
  "messages":[
    "Server machine 'https://ip-addr:7443/arcgis/sharing/rest/community/self' returned an error. 'Invalid token.'"
  ],
  "code":498
}
```

**Comment by kevinmichaelchen** [comment]
We're hitting `/arcgis/portaladmin/security/users/createUser` with a token generated from `/arcgis/sharing/rest/generateToken`... This isn't right... We should use a token generated from `/arcgis/portaladmin`.

**Comment by kevinmichaelchen** [comment]
It's a shot in the dark, but it might have something to do with certain characters in passwords (e.g., `!`)

**Comment by kevinmichaelchen** [comment]
Closed in #9
